### DISCUSSION TITLE
(FIXED kinda)The bot keeps refusing my request, I think It due to my school name.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#17760] [GitHub Education]
I'm a student of Alwin International public school padappai, sent like 5 request already, Everyone of them has been denied. When I give my school email, It gives out the wrong school name. This is due to zion being a "Group of schools" and most schools in the group (including mine) get the name zionschool.ac.in. Contact me on dhilipan.4472@zionschool.ac.in for more info.

**Comment by mlarasusan** [comment]
Hey [ArchflexTech](https://github.com/ArchflexTech) - it says "fixed kinda", were you able to fix it? Let me know, so I can help you :]

**Reply by ghost** [reply]
I applied using the "If your school is not here" feature, But I have the student developer pack on my personal email now. That's fine with me, But still my school should give different email for different schools and not give it to every school as a whole. Anyways, I got my student developer pack and now I enjoy using it! 

Note: I found out that the bot checks for the school name in your proof and if any other name is found, It gets auto rejected. But if name is found, It gets to verification by staff stage.
